saleor/graphql/checkout/tests/mutations/test_checkout_complete_with_payment.py

```diff
@@ -2364,16 +2364,36 @@ def test_checkout_complete_product_on_old_sale(
     )
 
 
+@pytest.mark.parametrize(
+    (
+        "use_legacy_voucher_propagation",
+        "expected_voucher_discount_value_type",
+        "expected_voucher_discount_value",
+    ),
+    [
+        (True, DiscountValueType.FIXED, Decimal("1.5")),
+        (False, DiscountValueType.PERCENTAGE, Decimal(10)),
+    ],
+)
 def test_checkout_with_voucher_on_specific_product_complete_with_product_on_promotion(
```

Comment:
Maybe worth to also adjust other voucher related tests, ie: 
-  `test_checkout_with_voucher_on_specific_product_complete`
- `test_checkout_complete_with_voucher_apply_once_per_order`
 At least add an assertion that it uses legacy flow